style.css

```diff
@@ -1,336 +1,742 @@
-  margin-left: 26px;
-  margin-right: 22px;
-  margin-bottom: 30px;
-  border: 1px solid #d0d9d4;
-  border-radius: 8px;
-  padding-bottom: 24px;
-}
-
-.multi {
-  background-color: white;
-}
-
-.image-grey {
-  width: 327px;
-  height: 252px;
-}
-
-.about-me-title {
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  color: #172b4d;
-  font-family: 'Inter', sans-serif;
-  margin-bottom: 0;
-}
-
-.span-width {
-  margin: 0 20px;
-}
-
-.about-me-paragraph {
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: normal;
-  font-size: 20px;
-  line-height: 28px;
-  text-align: center;
-  color: #172b4d;
-  width: 351px;
-  margin-top: 0;
-  padding-left: 10px;
-}
-
-.about-cont {
-  display: flex;
-  justify-content: space-between;
-}
-
-.about-me {
-  background-image: url(images/about.png);
-  background-repeat: no-repeat;
-  background-position-y: center;
-}
-
-.get-my-resume {
-  background: #36b37f;
-  border-radius: 4px;
-  width: 159px;
-  height: 48px;
-  top: 12px;
-  left: 12px;
-  font-family: 'Inter', sans-serif;
-  color: white;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  margin-top: 38px;
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-  border: none;
-  margin-bottom: 50px;
-}
-
-.get-my-resume:active {
-  background: #008552;
-  font-weight: bold;
-}
-
-.container-skills {
-  width: 327px;
-  height: 360px;
-  border-radius: 8px;
-  background-color: #ebf0ee;
-  margin-left: 24px;
-  margin-bottom: 24px;
-}
-
-.languages-title {
-  color: #344563;
-  font-family: 'Inter', sans-serif;
-  text-align: center;
-}
-
-.icon-lang {
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-  padding-top: 40px;
-}
-
-.data {
-  background-image: url(images/form.svg);
-  background-repeat: no-repeat;
-  background-position: right;
-}
-
-.final-text {
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  color: #172b4d;
-  font-family: 'Inter', sans-serif;
-  font-weight: bold;
-  margin-bottom: 58px;
-}
-
-.input-name,
-.email-address {
-  width: 327px;
-  height: 48px;
-  margin: 0 22px 24px 24px;
-  padding-left: 10px;
-  border-radius: 4px;
-  border: 1px solid #d0d9d4;
-  box-sizing: border-box;
-}
-
-.text1 {
-  width: 327px;
-  height: 114px;
-  margin: 20px 22px 24px 24px;
-  border: 1px solid #60c095;
-  box-sizing: border-box;
-  border-radius: 4px;
-  padding: 10px 0 0 10px;
-  color: #091e42;
-}
-
-.final-email {
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: normal;
-  font-size: 20px;
-  line-height: 32px;
-  text-align: center;
-  color: #42526e;
-  height: 375px;
-}
-
-.side-form {
-  position: absolute;
-}
-
-.icons-skills,
-.icons-frame,
-.icons {
-  margin-left: 8px;
-  color: #36b37e;
-  background: white;
-  border: none;
-  border-radius: 8px;
-  height: 40px;
-  margin-top: 10px;
-  font-size: 12px;
-  letter-spacing: 0.03em;
-  font-weight: 600;
+@media only screen and (min-width: 600px) {
```

Comment:
It's important that we use shorter ranges in our queries to have more control over the final output of the elements and their design 👀 We should be working with desktop sizes which normally goes around 1024 and beyond. Please be sure to use a media query whose breakpoint value oscillates between 992px and 1100px. This will improve your desktop design quite a lot and make your project meet the requirement 🤓 By making this modification in your code, you will also address a lot of positioning issues that you have now. 

![image](https://user-images.githubusercontent.com/15807343/154576830-285b58a9-ae6f-4ccb-a77d-896ccf713144.png)

![image](https://user-images.githubusercontent.com/15807343/154576871-e609e89a-b509-4f46-9867-4f7a83a516be.png)

It will make your life easier and much more enjoyable once you work within a shorter range of sizes 😁 Because right now your desktop size is anything bigger than 600px and controlling the elements that way is really hard

Reply:
Sorry Mateo I forgot that in the requirements is written that the width should be between 992px and 1100px, I will submit my work again sorry